frontend/src/containers/Header/Header.module.scss

```diff
@@ -2,8 +2,8 @@
 @import '/src/styles/mixins';
 
 .headerWrapper {
-  padding-top: 30px;
-  @include flex(row, center, center, nowrap);
+  padding: 16px;
+  @include flex(row, space-between, center, nowrap);
```

Comment:
It unnecessary adding all props for row flexbox, you may use get only "row" and "space-between" and other will be save like customs